### ISSUE TITLE
Geckodriver

### ISSUE THREAD

**Issue by psuhaj** [astaruch/slovakrail-ticket-script#1]
This  package requires geckodriver (at least on my system - ubuntu 16.04). May you please add it to description? 

Steps:
```
wget https://github.com/mozilla/geckodriver/releases/download/v0.24.0/geckodriver-v0.24.0-linux64.tar.gz
tar -xvzf geckodriver-v0.24.0-linux64.tar.gz
chmod +x geckodriver
sudo mv geckodriver /usr/local/bin
```

**Comment by astaruch** [comment]
Hi, there was pull request #2 which has fixed this.

**Comment by psuhaj** [comment]
Yes I know, we made it with @europ. Thanks for merge. :)

**Comment by europ** [comment]
I'm sorry.
I forgot to add `Closes #1` to the #2 PR description. I added it now.
